packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Providers/EnumProvider.cs

```diff
@@ -46,6 +57,15 @@ protected override TypeProvider[] BuildSerializationProviders()
         protected override string GetNamespace() => CodeModelPlugin.Instance.Configuration.ModelNamespace;
 
         protected override bool GetIsEnum() => true;
+        protected override TypeSignatureModifiers GetDeclarationModifiers()
+        {
+            var modifiers = TypeSignatureModifiers.Enum;
+            if (_inputType.Accessibility == "internal")
+            {
+                modifiers |= TypeSignatureModifiers.Internal;
+            }
+            return modifiers;
+        }
```

Comment:
do we really need this? because both of its derived types have overridden of this method, this should never be called I think